### ISSUE TITLE
Screen reader is not announcing descriptive information on the pager section in windows.

### ISSUE THREAD

**Issue by msftedad** [telerik/kendo-react#3477]
### Description

**User Impact:**
Screen reader users will miss important descriptive details on the pager section, making navigation unclear.
This can lead to confusion and difficulty in understanding page context or available actions.

### Plan

KendoReact

### Steps To Reproduce

1. Open the URL: https://www.telerik.com/kendo-react-ui/components/grid in edge browser in windows.
2. 'React Data Grid Overview' page will be open.
3. Press tab key to navigate to the pager section.
4. Verify whether screen reader is announcing descriptive information on the pager section in windows or not.

### Screenshots or video

**Refer Attachment:**

1. Screen reader is not announcing descriptive information on the pager section in windows.png
2. Screen reader is not announcing descriptive information on the pager section in windows.mp4

<img width="974" height="530" alt="Image" src="https://github.com/user-attachments/assets/f24577ef-7b1d-476e-b1c8-6627eeb1830a" />
https://github.com/user-attachments/assets/7330e56c-9d9b-49c5-b17b-30df7fff9e35

### Actual Behavior

Screen reader is not announcing descriptive information on the pager section in windows.
screen reader announces as 'Page 1 of 10 pager'

### Expected Behavior

Screen reader should announce descriptive information on the pager section in windows.

### Browser

Chrome, Microsoft Edge

### Browser version

Edge (Version 143.0.3650.46)

### OS type

MacOS, Windows

### OS version

Windows 11 Version 25H2 (OS Build 26220.7271)

### Last working version of the KendoReact package (if regression).

No
